### DISCUSSION TITLE
Create Dynamic Open Graph Images with Cloudinary — Jagad Yudha Awali

### DISCUSSION THREAD

**Discussion by giscus** [jagadyudha/jagad.dev#102] [Announcements]
# Create Dynamic Open Graph Images with Cloudinary — Jagad Yudha Awali

Make dynamic Open Graph images with the help of Cloudinary URL transformations

https://jagad.dev/posts/dynamic-open-graph-cloudinary

**Comment by claudioripoli** [comment]
Hi! I tried to reach out to you on polywork.com. If you have some time, please take a look. 
Thank you, Claudio.
